src/Symfony/Component/Messenger/Command/AbstractFailedMessagesCommand.php

```diff
@@ -29,20 +30,25 @@
  */
 abstract class AbstractFailedMessagesCommand extends Command
 {
-    private $receiverName;
-    private $receiver;
+    private $globalFailureReceiverName;
+    private $failureTransports;
 
-    public function __construct(string $receiverName, ReceiverInterface $receiver)
+    public function __construct(?string $globalFailureReceiverName, $failureTransports)
     {
-        $this->receiverName = $receiverName;
-        $this->receiver = $receiver;
+        $this->failureTransports = $failureTransports;
+        $this->globalFailureReceiverName = $globalFailureReceiverName;
+        if (!$failureTransports instanceof ServiceLocator) {
+            trigger_deprecation('symfony/messenger', '5.2', 'Passing failureTransports should now pass a ServiceLocator', __METHOD__);
```

Comment:
5.3 now - same as anywhere else in this PR

For the wording, check out these examples:

https://github.com/symfony/symfony/blob/7754beddfc6be3a66aa5dc561051efabd2aaee8f/src/Symfony/Component/HttpFoundation/InputBag.php#L33